src/Button/index.js

```diff
@@ -170,9 +171,7 @@ export const StyledButton = styled(
       `};
   }
 
-  &:focus {
-    ${getButtonBoxShadow(BUTTON_STATES.FOCUS)};
-  }
+  ${getFocus()}
```

Comment:
It could be used just like `${getFocus}` if you delete the first empty arrow function, but it's up to you 🙂